hindi/Bhagat Singh/BhagatSingh.md

```diff
@@ -0,0 +1,3 @@
+Freedom Fighter
+
+To know about real india hero click [Bhagat Singh](Bhagat_Singh/bhagat_singh.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.